web-client/src/client/lib.rs

```diff
@@ -443,13 +445,10 @@ impl Client {
     #[wasm_bindgen(js_name = getBlock)]
     pub async fn get_block(&self, hash: &str) -> Result<PlainBlockType, JsError> {
         let hash = Blake2bHash::from_str(hash)?;
-        let block = self
-            .inner
-            .consensus_proxy()
-            .blockchain
-            .read()
-            .get_block(&hash, false)?;
-        Ok(serde_wasm_bindgen::to_value(&PlainBlock::from_block(&block))?.into())
+        let blockchain = self.inner.consensus_proxy().blockchain;
+        let blockchain = blockchain.read();
+        let block = blockchain.get_block(&hash, false)?;
```

Comment:
I'd have left it as it was

Reply:
https://github.com/nimiq/core-rs-albatross/pull/3550#discussion_r2588096374

Reply:
Ah yes, sorry. Didn't spot that